### ISSUE TITLE
Unable to filter the count using attributeExists on its own

### ISSUE THREAD

**Issue by Tedsterh** [multimeric/AmplifyCountDirective#8]
Hi,

I am trying to filter based on wether an attribute exists or not, if I use an and statement alongside the `attributeExists` I am able to get a count but on its own it throws and error
```json
"errors": [
    {
      "path": [
        "fuseable"
      ],
      "data": null,
      "errorType": "Lambda:Unhandled",
      "errorInfo": null,
      "locations": [
        {
          "line": 2,
          "column": 3,
          "sourceName": null
        }
      ],
      "message": "ExpressionAttributeValues must not be empty"
    }
  ]
```

**Comment by multimeric** [comment]
Can you show me the schema and the query that you used? You can adjust type names if you want to keep any of it private, but I think I need to know the structure.

**Comment by Tedsterh** [comment]
`type Trait @model @searchable @count {
    id: ID!
    trait: String! @default(value: "trait")
    name: String! @index(name: "traitByName", queryField: "traitByName", sortKeyFields: ["animalId"])
    animalId: ID @index(name: "traitByAnimal", queryField: "traitByAnimal", sortKeyFields: ["createdAt"])
    animal: Animal @belongsTo(fields: ["animalId"])
    createdAt: AWSDateTime!
}
`

so I am trying to filtering the count based on how many traits have an animal id, luckily I already have a default string on every trait so that I could use sorting on amplify indexes, but I have to have this extra filter `{trait: {eq: "trait"}}` alongside the `{animalId: {attributeExists: true}}`

Hope this helps

**Comment by multimeric** [comment]
So you are saying your overall filter is this?
```graphql
{
  countTrait(filter: {
    and: {
      trait: {eq: "trait"},
      animalId: {attributeExists: true}}
  })
}
```

Can you run this query directly in the AppSync console (https://us-east-1.console.aws.amazon.com/appsync/home) and let me know if it gives you the same response?

**Comment by Tedsterh** [comment]
Yes that's the query sorry, I can run that one fine but as soon as I remove the trait filter and leave just the attributeExists I get the error

**Comment by multimeric** [comment]
Fixed in 1.1.0.
